cupy/core/core.pyx

```diff
@@ -1621,30 +1621,63 @@ cdef class ndarray:
                 The default uses CUDA stream object of the current context.
             order ({'C', 'F', 'A'}): The desired memory layout of the host
                 array. When ``order`` is 'A', it uses 'F' if the array is
-                fortran-contiguous and 'C' otherwise.
+                fortran-contiguous and 'C' otherwise. The ``order`` will be
+                ignored if ``out`` is specified.
+            out Output array. It should be a pinned memory to enable
```

Comment:
Please fix markup. We should specify the type as `numpy.ndarray`.

The sentence should be something like this:
"In order to enable asynchronous copy, the underlying memory should be a pinned memory."